pytorch_lightning/trainer/trainer.py

```diff
@@ -314,6 +315,12 @@ def __init__(
             terminate_on_nan: If set to True, will terminate training (by raising a `ValueError`) at the
                 end of each training batch, if any of the parameters or the loss are NaN or +/-inf.
 
+                .. deprecated:: v1.5
```

Comment:
there is no transition from the old argument to the new one like if "anomaly" is not set and "termination" yes the use "termination" value as "anomaly"?